### PR TITLE
lantiq: xrx200: fix use after free bug

### PR DESCRIPTION
**Pull request by abajk**
The skb->len field is read after the packet is sent to the network
stack. In the meantime, skb can be freed. This patch fixes this bug.

Signed-off-by: Aleksander Jan Bajkowski <olek2@wp.pl>

### CODE REVIEW

**Comment by jospezial** [comment]
0713-v5.17-MIPS-lantiq-dma-increase-descritor-count.patch
descritor is portuguese for descriptor.

File: target/linux/lantiq/patches-5.10/0715-v5.17-net-lantiq_xrx200-convert-to-build_skb.patch [hunk 1]
```diff
@@ -1,7 +1,7 @@
 From e015593573b3e3f74bd8a63c05fa92902194a354 Mon Sep 17 00:00:00 2001
 From: Aleksander Jan Bajkowski <olek2@wp.pl>
 Date: Tue, 4 Jan 2022 16:11:44 +0100
-Subject: [PATCH 715/715] net: lantiq_xrx200: convert to build_skb
+Subject: [PATCH] net: lantiq_xrx200: convert to build_skb
```
Comment (ynezz) [inline target/linux/lantiq/patches-5.10/0715-v5.17-net-lantiq_xrx200-convert-to-build_skb.patch:4]: Remove all this unrelated changes, thanks.

**Review by ynezz — CHANGES_REQUESTED** [review]

File: target/linux/lantiq/patches-5.10/0715-v5.17-net-lantiq_xrx200-convert-to-build_skb.patch (hunk 1 above)
Comment (abajk) [reply target/linux/lantiq/patches-5.10/0715-v5.17-net-lantiq_xrx200-convert-to-build_skb.patch:4]: My intention was to add the kernel version to the patch name.

Comment (ynezz) [reply target/linux/lantiq/patches-5.10/0715-v5.17-net-lantiq_xrx200-convert-to-build_skb.patch:4]: That's ok, but we don't need that subject change, do we?

**Comment by ynezz** [comment]
Thanks! Pulled into my staging tree at https://git.openwrt.org/openwrt/staging/ynezz.git

**Comment by jospezial** [comment]
https://github.com/openwrt/openwrt/commit/c46621b3f3fff89cce4dd5e24bd52a4337e44378

**Pull request closed without merging**